e2e/libs/keywords/statefulset_keywords.py

```diff
@@ -0,0 +1,141 @@
+from workload.persistentvolumeclaim import delete_persistentvolumeclaim
+from workload.statefulset import create_statefulset
+from workload.statefulset import delete_statefulset
+from workload.statefulset import get_statefulset
+from workload.statefulset import scale_statefulset
+from workload.statefulset import wait_for_statefulset_replicas_ready
+from workload.workload import check_pod_data_checksum
+from workload.workload import get_workload_pod_names
+from workload.workload import get_workload_pvc_name
+from workload.workload import get_workload_volume_name
+from workload.workload import keep_writing_pod_data
+from workload.workload import write_pod_random_data
+
+from persistentvolumeclaim.persistentvolumeclaim import PersistentVolumeClaim
+
+from volume.volume import Volume
+
+from volume.constant import MEBIBYTE
+
+from utility.resource_cache import ResourceCache
+
+from utility.constant import CACHE_CHECKSUM
+from utility.constant import CACHE_EXPANDED_SIZE
+from utility.constant import CACHE_ORIGINAL_REPLICA_COUNT
+from utility.constant import KIND_PERSISTENTVOLUMECLAIM
+from utility.constant import KIND_POD
+from utility.constant import KIND_STATEFULSET
+from utility.constant import KIND_VOLUME
+
+
+class statefulset_keywords:
+
+    def __init__(self):
+        self.volume = Volume()
+        self.persistentvolumeclaim = PersistentVolumeClaim()
+
+    def cleanup_statefulsets(self):
```

Comment:
Is there a way to configure `Build with Parameter` so I could run the test in pipeline using the PR by myself?

Reply:
Running locally seems to pass:
```
Tests.Kubelet Restart :: Negative Test Cases                          | PASS |
1 test, 1 passed, 0 failed
```
Could you trigger the pipeline test again with the trace log?

Reply:
The pipeline has set to based on `https://github.com/c3y1huang/longhorn-tests` 's `refactor-negative-variable-list` branch.

Reply:
Looking at the test report, it errored out earlier in [Restart Workload Kubelet ${workload_name}, ${duration}](https://ci.longhorn.io/job/private/job/longhorn-e2e-test/114/robot/report/log.html#s1-s2-t1-k3-k1-k2-k1).

Reply:
@yangchiu https://ci.longhorn.io/job/private/job/longhorn-e2e-test/115/ seems to be stuck. Could you help to check this?

Reply:
I am having no luck reproducing this locally.
[test-report.zip](https://github.com/longhorn/longhorn-tests/files/13496477/test-report.zip)

Reply:
Pipeline test passed https://ci.longhorn.io/job/private/job/longhorn-e2e-test/123/robot/report/log.html

I suspect that https://ci.longhorn.io/job/private/job/longhorn-e2e-test/114/ failed somewhere during kubelet restart. leading to cascading failures because statefulset couldn't get removed.

Reply:
> Could you trigger the pipeline test again with the trace log?

This is another thing needs to be improved. Console log and test report should be enough to know what happened during the test, enable trace log and rerun the test again cause duplicated and redundant efforts to debugging.

Currently adding message to assert to improve this.

Reply:
> Could you trigger the pipeline test again with the trace log?

This is another thing needs to be improved. Console log and test report should be enough to know what happened during the test, enable trace log and rerun the test again cause duplicated and redundant efforts to debugging.

Currently adding message to assert to improve this.